packages/github-actions/teams-toolkit-cicd/__tests__/operations.test.ts

```diff
@@ -0,0 +1,115 @@
+import * as fs from 'fs-extra'
+import mock from 'mock-fs'
+import {Operations} from '../src/operations'
+import {BaseError} from '../src/base-error'
+import * as path from 'path'
+import CommandUtil from '../src/utils/exec'
+
+const projectRoot = '/home/test/'
+
+describe('Test BuildTeamsApp', () => {
+  beforeEach(() => {
+    mock({
+      '/home/test': {
+        '.fx': {
+          'env.default.json': JSON.stringify({})
+        }
+      }
+    })
+  })
+  afterEach(() => {
+    mock.restore()
+  })
+  test('[BuildTeamsApp] LanguageError', async () => {
+    expect.assertions(1)
+    // Arrange
+    // Act & Assert
+    try {
+      await Operations.BuildTeamsApp(projectRoot, ['tabs'])
+    } catch (error) {
+      expect(error).toBeInstanceOf(BaseError)
+    }
+  })
+
+  test('[BuildTeamsApp] Happy path', async () => {
+    expect.assertions(0)
+    // Arrange
+    await fs.writeJson(path.join(projectRoot, '.fx', 'env.default.json'), {
+      solution: {
+        programmingLanguage: 'javascript'
+      }
+    })
+
+    // Act & Assert
+    try {
+      await Operations.BuildTeamsApp(projectRoot, ['tabs'])
+    } catch (error) {
+      expect(true).toBeFalsy() // Should not reach here.
```

Comment:
It is strange here, what's the purpose of this code? do we expect it will not throw error here?